samples/ingestion/ingestion-client/FetchTranscription/TranscriptionProcessor.cs

```diff
@@ -319,14 +319,14 @@ private async Task ProcessSucceededTranscriptionAsync(string transcriptionLocati
             log.LogInformation($"Got succeeded transcription for job {jobName}");
 
             var textAnalyticsKey = FetchTranscriptionEnvironmentVariables.TextAnalyticsKey;
-            var textAnalyticsRegion = FetchTranscriptionEnvironmentVariables.TextAnalyticsRegion;
+            var textAnalyticsEndpoint = FetchTranscriptionEnvironmentVariables.TextAnalyticsEndpoint;
             var textAnalyticsInfoProvided = !string.IsNullOrEmpty(textAnalyticsKey)
-                && !string.IsNullOrEmpty(textAnalyticsRegion)
-                && !textAnalyticsRegion.Equals("none", StringComparison.OrdinalIgnoreCase);
+                && !string.IsNullOrEmpty(textAnalyticsEndpoint)
+                && !textAnalyticsEndpoint.Equals("none", StringComparison.OrdinalIgnoreCase);
```

Comment:
nit: please remove the equals none check - it was only there since the default for region is "none"